safe-global/web-core
# feat: basic message signing modal

## What it solves

Partially resolves #1206

## How this PR fixes it

A `MsgModal` has been created that opens when clicking "Sign" on mock messages. Template functionality for dispatching of `MsgEvent`s has also been included for extending when the endpoints are in place.

## How to test it

Click "Sign" on mock messages in the "Messages" area of the transaction list and observe the new modal which matches [the designs](https://www.figma.com/file/x1uuIJsOZkyv5cwNlDdVC1/EIP-1271?node-id=256%3A5320&t=fc6PHbsj7PrgdzLt-0).

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/203366565-d19f72e0-0bca-4416-86a1-0de90419332c.png)
